pyproject.toml

```
@@ -307,9 +311,9 @@ test = ["test-parallel", "test-cflakes"]
 test-slow = "pytest -vvv tests"
 coverage = "pytest --cov=wetterdienst tests"
 coverage-parallel = "pytest --cov=wetterdienst --numprocesses=auto -m 'not (explorer or cflake)' tests"
-export_licenses = "pip-licenses --format=plain-vertical --with-authors --with-urls --with-license-file --no-license-path --ignore-packages wetterdienst --output-file=THIRD_PARTY_NOTICES"
-export = ["export_licenses"]
+export = "pip-licenses --format=plain-vertical --with-authors --with-urls --with-license-file --no-license-path --ignore-packages wetterdienst --output-file=THIRD_PARTY_NOTICES"
```

`poe export` would be a bit poor to figure out what it actually _means_. Maybe use a different keyword for that task?
```suggestion
licenses = "pip-licenses --format=plain-vertical --with-authors --with-urls --with-license-file --no-license-path --ignore-packages wetterdienst --output-file=THIRD_PARTY_NOTICES"
```